cwl/src/main/scala/cwl/Workflow.scala

```diff
@@ -25,7 +25,9 @@ case class Workflow private(
                      id: String,
                      inputs: Array[InputParameter],
                      outputs: Array[WorkflowOutputParameter],
-                     steps: Array[WorkflowStep]) {
+                     steps: Array[WorkflowStep],
+                     requirements: Option[Array[Requirement]],
+                     hints: Option[Array[CwlAny]]) {
```

Comment:
These must also be added to `ExpressionTool`

Reply:
Strangely enough, they are already there (I did not add them). 😮

Reply:
Huh, I found my CNil was due to an errant `ExpressionTool`, I guess it wasn't the requirement that it wasn't liking after all 😨

Reply:
Unit test or Centaur? Unit tests currently are not SALADing so you have to format requirements just so.